quodlibet/po/fr.po

```diff
@@ -5661,156 +5622,147 @@ msgid "Display remaining time"
 msgstr "Montrer la durée restante"
 
 #: ../quodlibet/qltk/shortcuts.py:16
-#, fuzzy
 msgid "Main Window"
-msgstr "Paroles"
+msgstr "Fenêtre principale"
 
 #: ../quodlibet/qltk/shortcuts.py:17
 msgid "Seek backwards by 10 seconds"
-msgstr ""
+msgstr "Reculer de 10 secondes"
 
 #: ../quodlibet/qltk/shortcuts.py:18
 msgid "Seek forward by 10 seconds"
-msgstr ""
+msgstr "Avancer de 10 secondes"
 
 #: ../quodlibet/qltk/shortcuts.py:19
-#, fuzzy
 msgid "Focus the search entry"
-msgstr "Focaliser le lecteur en fonction"
+msgstr "Focaliser sur la requête de recherche"
 
 #: ../quodlibet/qltk/shortcuts.py:22
-#, fuzzy
 msgid "Reset filters and jump to the playing song"
-msgstr "Se déplacer dans la chanson en cours de lecture"
+msgstr "Réinitialiser les filtres et aller au titre en cours de lecture"
 
 #: ../quodlibet/qltk/shortcuts.py:26
 msgid "Open the information window for the selected songs"
-msgstr ""
+msgstr "Ouvrir la fenêtre d'informations des titres sélectionnés"
 
 #: ../quodlibet/qltk/shortcuts.py:27
 msgid "Open the tag editor for the selected songs"
-msgstr ""
+msgstr "Ouvrir le gestionnaire d'étiquettes des titres sélectionnés"
 
 #: ../quodlibet/qltk/shortcuts.py:28
-#, fuzzy
 msgid "Queue the selected songs"
-msgstr "Impossible de supprimer les chansons"
+msgstr "Placer en file de lecture les titres sélectionnés"
 
 #: ../quodlibet/qltk/shortcuts.py:29
-#, fuzzy
 msgid "Delete the selected songs"
-msgstr "Impossible de supprimer les chansons"
+msgstr "Supprimer les titres sélectionnés"
 
 #: ../quodlibet/qltk/shortcuts.py:30
 msgid "Show the inline search entry"
-msgstr ""
+msgstr "Afficher le contenu de la recherche interne"
```

Comment:
Maybe we should have a look at source code to understand the meaning. "Show the inline search entry" means : show what is entered as a query, i.e. "Afficher le contenu de la recherche interne". Other wize English should be corrected in source.